app/decorators/proposal_decorator.rb

```diff
@@ -31,6 +31,20 @@ def approvals_in_list_order
     end
   end
 
+  def subscribers_list
+    userXrole = object.users.map {|u| [u, Role.new(u, object)] }
+    schwartzian_uXrXo = userXrole.map do |user, role|
```

Comment:
This graphic was meant to convey "I have no idea what that variable means" :blush: